NitroxClient/Persistence/Model/PersistedClientDataModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NitroxClient.Persistence.Model
+{
+    [Serializable]
```

Comment:
I tried making this a ProtobufContract/JsonObject like on the server, but the client stopped working after I did, so I had to remove it.

Reply:
Json (at least the version from the server) can't be used in Client and Model because Oculus is using a custom (and old) version which is interfering.

Reply:
You could use LitJson if Newtonsoft isn't working for your needs.

Reply:
I do not like the idea of pulling in yet another JSON library to the project. That sounds reductive to future maintainers as they will have to support 2 JSON implementations (asinine). I instead suggest we get Newtonsoft  working somehow, I just need to figure out why my current solution is not working on the client... I feel it has something to do with the File class API on Windows not working right injected in to the client...

Reply:
LitJson is already used in Subnautica and Nitrox for Asset handling

Reply:
🤔 I will try it and see what I come up with, but I feel that if the (de)SerializeObject functions weren't working, there would just be a blank file on the client's disk, where there is not... I feel the point of failure is at the File class function calls...

Reply:
Through some digging on the documentation, I found there isn't similar behaviour to Newtonsoft that just simply takes a C# object and returns a string and vice versa. https://litjson.net/api/LitJson/JsonWriter/ I would have to write a custom implementation of JsonWriter for each and every function call, and expecting the implementer in the future to do that too might be asking too much...

Reply:
I confirmed my theory, if you just replace the Desearlize calls with a blank string, the file still does not get created... It's clearly a problem with the client not working with files correctly, I just have 0 idea what's causing it...